packages/desktop-client/src/components/modals/CloseAccountModal.tsx

```diff
@@ -142,10 +151,14 @@ export function CloseAccountModal({
               {balance !== 0 && (
                 <View>
                   <Paragraph>
-                    This account has a balance of{' '}
-                    <strong>{integerToCurrency(balance)}</strong>. To close this
-                    account, select a different account to transfer this balance
-                    to:
+                    <Trans>
+                      This account has a balance of{' '}
+                      <strong>
+                        <Trans>{{ balance: integerToCurrency(balance) }}</Trans>
```

Comment:
I don't think we should use nested `Trans` here, is it necessary?

Reply:
I'm having trouble using the object literal inside another tag, do you know a better way round it? It won't let me use it unless it's in the first level of the Trans element